src/SlashCommands.tsx

```diff
@@ -472,14 +475,18 @@ export const Commands = [
                 );
             }
 
-            const topicEvents = room.currentState.getStateEvents('m.room.topic', '');
-            const topic = topicEvents && topicEvents.getContent().topic;
-            const topicHtml = topic ? linkifyAndSanitizeHtml(topic) : _t('This room has no topic.');
+            const content: MRoomTopicEventContent = room.currentState.getStateEvents('m.room.topic', '')?.getContent();
+            const topic = !!content ? ContentHelpers.parseTopicContent(content)
+                : { text: _t('This room has no topic.') };
```

Comment:
Code style changes are inbound for this sort of thing:
```suggestion
            const topic = !!content 
                ? ContentHelpers.parseTopicContent(content)
                : { text: _t('This room has no topic.') };
```